torch/nn/parallel/distributed.py

```diff
@@ -224,6 +224,12 @@ def __getstate__(self):
         return attrs
 
     def __setstate__(self, state):
+        # This is done to ensure proper functioning for states saved prior to the removal of __hash__
+        # for tensors / parameters
+        if 'bucket_map' in state.keys():
+            for p in state['bucket_map'].keys():
+                state['bucket_map'][id(p)] = state['bucket_map'].pop(p)
```

Comment:
Actually is it even possible to load a dict of tensors now that they are unhashable?